src/elasticsearch/score.ts

```diff
@@ -4,6 +4,7 @@ export default function getFunctionScores (config) {
   }
   let filter = []
   let esScoringAttributes = config.elasticsearch.searchScoring.attributes
+  let esScoringConfig = config.elasticsearch.searchScoring
```

Comment:
Just to save like 5 characters ;)
```suggestion
  let esScoringConfig = config.elasticsearch.searchScoring
  let esScoringAttributes = esScoringConfig.searchScoring.attributes
```

Reply:
I think it should be like this 🤔 
```
  let esScoringConfig = config.elasticsearch.searchScoring
  let esScoringAttributes = esScoringConfig.attributes
```

Reply:
@alat Can you commit the suggested changes?